modules/apps/click-to-chat/click-to-chat-web/src/main/resources/META-INF/resources/dynamic_include/view.jsp

```diff
@@ -16,4 +16,70 @@
 
 <%@ include file="/dynamic_include/init.jsp" %>
 
-<liferay-util:include page='<%= "/dynamic_include/" + clickToChatChatProviderId + ".jsp" %>' servletContext="<%= application %>" />
\ No newline at end of file
+<liferay-util:include page='<%= "/dynamic_include/" + clickToChatChatProviderId + ".jsp" %>' servletContext="<%= application %>" />
+
+<style type="text/css">
+	.click-to-chat-hidden {
+		display: none !important;
+	}
+</style>
+
+<script type="text/javascript">
+    (function () {
+        function changeVisible(selectors, show) {
+            let selectorsList = selectors.split(',')
+            if(show) {
+                selectorsList.forEach((selector) => {
+                    document.querySelectorAll(selector).forEach(el => el.classList.add('click-to-chat-hidden'));
+                })
+            }else{
+                selectorsList.forEach((selector) => {
+                    document.querySelectorAll(selector).forEach(el => el.classList.remove('click-to-chat-hidden'));
+                })
+            }
+        }
+
+        let providers = {
+            'chatwoot': function (hide) {
+                if(hide) {
+                    document.querySelectorAll('.woot--bubble-holder,.woot-widget-holder').forEach(el => el.remove());
+                }
+            },
+            'crisp': '.crisp-client',
+            'hubspot': '#hubspot-messages-iframe-container',
+            'jivochat': '.globalClass_07d6',
+            'livechat': '#chat-widget-container',
+            'liveperson': '.LPMcontainer.LPMoverlay,.lp_desktop',
+            'smartsupp': '#chat-application',
+            'tawkto': function (hide) {
+                if(hide) {
+                    window?.Tawk_API?.hideWidget();
```

Comment:
I think this window I'll always exist, the conditional chaining could be applied after the first parameter, in this case window.Tawk_API?.hideWidget();